aws/resource_aws_appsync_datasource_test.go

```diff
@@ -188,6 +213,7 @@ resource "aws_appsync_datasource" "test" {
   api_id = "${aws_appsync_graphql_api.test.id}"
   name = "tf_appsync_%s"
   type = "AMAZON_DYNAMODB"
+  description = "appsync datasource version1"
```

Comment:
Nitpick: Generally its cleaner to parameterize the test configuration values when changing a single one instead of creating a mostly duplicated new test configuration, e.g. 

```go
func testAccAppsyncDatasourceConfig_ddb(rName, description string) string {
// ...
description = %q
// ...
`, rName, rName, rName, rName, rName, rName, description)
}
```